test/diags_test.py

```diff
@@ -106,7 +106,10 @@ def execute(self, test_str, imagefilename, imagethreshold, ncfiles, rtol, atol):
         proc = subprocess.Popen([self.diagstr], shell=True)
         proc_status = proc.wait()
         if proc_status!=0: 
-            raise DiagError("diags run failed")
+            proc = subprocess.Popen([self.diagstr], shell=True)
+            proc_status = proc.wait()
+            if proc_status!=0: 
+                raise DiagError("diags run failed")
```

Comment:
@painter1 why do we do this twice now? Is it to bypass @jccosta issue were sometimes we get "bad template" error?